daemon/logger/journald/journald.go

```diff
@@ -0,0 +1,30 @@
+package journald
+
+import (
+	"github.com/coreos/go-systemd/journal"
+	"github.com/docker/docker/daemon/logger"
+)
+
+type Journald struct {
+	Jmap map[string]string
+}
+
+func New(id string) (logger.Logger, error) {
+	jmap := map[string]string{"MESSAGE_ID": id}
```

Comment:
Makes sense to check here `journal.Enabled()`. And then your returned error will be used :)